### PR TITLE
Rk/move email functions to backend

### PR DESCRIPTION
**Pull request by rkaufman13**
The BULK of this PR involves removing the AWS logic from the frontend and calling a server instead.
Please note that going forward, if you do no additional setup, you will get console errors when triggering an action that sends an email (such as creating an event with an email address provided). These can safely be ignored. However, to get rid of them and test the end to end flow entirely, you will need to clone the `noodle-backend` repo, run it, set the `REACT_APP_BACKEND_BASE_URL` to localhost:3001, and add localhost:3001 as an allowed host in the CORS allowlist.
Or you can just get the console errors. :)

Also in this PR, because I'm a mess:
-Deleting a lot of the `default` bits in custom.scss, and adding the logic that creates utility classes for our colors
-Replacing unneeded `row`s with `div`s on the Admin page
-Adding Headings on the alerts
-A ton of Prettier stuff that doesn't seem to matter (replacing single quotes with double, etc).

### CODE REVIEW

File: src/aws_helpers/index.js [hunk 1]
```diff
@@ -32,14 +20,19 @@ export const sendConfirmationEmail = (vars) => {
...
 
-  var sendPromise = ses.sendTemplatedEmail(params).promise();
-  sendPromise
-    .then(function (data) {
-      console.log(data);
-    })
-    .catch(function (err) {
-      console.error(err, err.stack);
-    });
+  console.log(JSON.stringify(params));
```
Comment (eggchan) [inline src/aws_helpers/index.js:23]: We've got a rogue console log, captain.

File: src/About.jsx [hunk 2]
```diff
@@ -1,15 +1,34 @@
...
-    return (
-        <Container><Row><Col>
-            <h2>About Noodle</h2>
-            
-            <p>Scheduling events <em>used</em> to be easy on the other sites. Then they started to suck. Intrusive ads, pop-ups, forced logins, slow-loading pages, all that jazz.</p>
+export const About = () => {
+  return (
+    <Container>
+      <Row>
+        <Col>
```
Comment (eggchan) [inline src/About.jsx:8]: I believe the `<Container>`, `<Row>`, and `<Col>` that this is inside lives in the App.js file, so you shouldn't need these tags.

File: scss/custom.scss
Comment (eggchan) [inline scss/custom.scss:1]: This is going to be fun for me to merge with the branch I'm working on haha

**Review by eggchan — CHANGES_REQUESTED** [review]

**Comment by rkaufman13** [comment]
So AWS's trust and safety team has denied my request to use them in prod, so I have to close this entire PR and start over

**Pull request closed without merging**